src/prettyPrint.js

```diff
@@ -0,0 +1,19 @@
+const prettyPrint = days => {
+  const numberOfMonths = Math.floor(days / 30);
+
+  if (numberOfMonths >= 1) {
```

Comment:
You could replace this with `days >= 30` to avoid making two rounding operations